content/agent/faq/agent-configuration-file-migration.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Agent configuration file migration
+kind: faq
+further_reading:
+---
+
+[If you haven't done it already, upgrade your Agent v5 to Agent v6][1]
+
+To automatically transition Agent configuration paths and formats from Agent v5 to Agent v6, use the `import` command. The command parses an existing v5 `datadog.conf` and converts the configuration options to the new v6 `datadog.yaml` format. It also copies configuration files for checks that are currently enabled.
```

Comment:
Instead of having a page that's specific to the `import` command, I feel this should be part of the `upgrade` page, otherwise it's a bit confusing because the `upgrade` page describes 2 methods:
1. the `one-step upgrade`, which takes care of running the `import` command as well, so for this method this page is irrelevant
2. the `manual upgrade`, which doesn't take care of running the command, so for this method the info on this page is actually relevant

We should add these instructions under the `manual upgrade` instructions.

Reply:
👌 good point